### ISSUE TITLE
Find a good way of setting a remote HEAD or dealing with lack of a HEAD

### ISSUE THREAD

**Issue by anishathalye** [anishathalye/git-remote-dropbox#1]
Currently, we don't ever set the remote HEAD. Normally, this is something that's done on the server side - the git client doesn't have a method of setting a remote HEAD.

This isn't a huge issue in terms of usability, but it's nice to have a HEAD so that a branch is automatically checked out after cloning.

It would be good to see if there's any way we can set a remote HEAD or deal gracefully with the lack of a HEAD.

Note: As the code is right now, it is possible to manually create the HEAD file (to point to, say, master), and everything will work as expected.

**Comment by tdy** [comment]
Just so I understand correctly, there's currently no way to make `git clone dropbox://myrepo.git` automatically switch to `master`, right? We have to use one of the following options each time:

```
git clone dropbox://myrepo.git -b master
```

```
git clone dropbox://myrepo.git
cd myrepo
git checkout master
```

**Comment by anishathalye** [comment]
Yes, with the current implementation, that is true.

**Comment by tdy** [comment]
Got it, thanks

**Comment by anishathalye** [comment]
It's mentioned as the last item in this list: https://github.com/anishathalye/git-remote-dropbox#notes

Were you thinking that it should be somewhere else in the readme?

**Comment by lukas-ais** [comment]
In the first section is written:

> To clone repositories in folders or shared folders mounted in your Dropbox, you can run:
> git clone "dropbox:///path/to/repo"

This what users are doing.

Why not directly guide to `git clone "dropbox:///path/to/repo" -b master`?

**Comment by anishathalye** [comment]
Sure, sounds reasonable. Updated.

**Comment by anishathalye** [comment]
Fixed in #84.
